src/styles/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  transition-duration: 300ms;
+  transition-property: background-color, color, border;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-family: $font;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $main-color;
+    padding-bottom: 4px;
+  }
+
+  &__productCode {
+    font-family: $font;
```

Comment:
Here again, the use of the `font` shorthand property would improve code readability and maintainability. For instance: `font: $fontWeightStandard 10px/14px $font;`.